priv/perf/apps/load_test/lib/service/faucet.ex

```diff
@@ -111,6 +129,13 @@ defmodule LoadTest.Service.Faucet do
     {:reply, {:ok, user_utxo}, updated_state}
   end
 
+  def handle_call({:merge_utxos, currency}, _from, %__MODULE__{faucet_account: faucet_account} = state) do
```

Comment:
```suggestion
  def handle_call({:merge_utxos, currency}, _from, state) do
```
and later use `state.faucet_account`

Reply:
it's just makes the code less bloated!